base_iso3166/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+ISO 3166 adaptation to Odoo
+===========================
+
+This module adds some data specified by ISO 3166, specifically:
+
+* ISO 3166-1 alpha-3 (3-char) codes in countries
+* ISO 3166-1 numeric codes in countries
+* ISO 3166-3 formerly used codes in countries
+
+Check https://en.wikipedia.org/wiki/ISO_3166 for more info.
+
+Installation
+============
+
+This module requires the Python library pycountry >= 0.19 installed in the
+system. It works also with pycountry >= 16.11.8
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/101/10.0
+
+Known issues / Roadmap
+======================
+
+* Include 3166-2 country subdivision codes
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/community-data-files/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```

Comment:
Remove last part of the link according latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst